From 00e45ebf2d862e5cebd269d57c52d8a8fc9527f0 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Mon, 5 Jan 2015 11:53:02 -0500
Subject: [redhat] kabi: Adding symbol efi (drivers/firmware/efi/efi.c)

Message-id: <7214ad905f542e981fe5148d53d7c925986fada9.1420451770.git.poros@redhat.com>
Patchwork-id: 102436
O-Subject: [PATCH RHEL7.1 05/24] kabi: Adding symbol efi (drivers/firmware/efi/efi.c)
Bugzilla: 1085963
RH-Acked-by: Kyle McMartin <kmcmarti@redhat.com>

Bugzilla: 1085963
    https://bugzilla.redhat.com/show_bug.cgi?id=1085963

    Description:
    ============
    Need this pointer for the iLo data structures.
    Used in hpdsa.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1070457 18/18] kabi: Adding symbol efi (arch/x86/platform/efi/efi.c)
 Jiri Olsa <jolsa@redhat.com> (Thu, 27 Feb 2014 10:03:32 +0100)
 Description:
 ============
 used to locate smbios records

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/efi

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/efi b/redhat/kabi/kabi-module/kabi_x86_64/efi
new file mode 100644
index 0000000..3da9270
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/efi
@@ -0,0 +1,2 @@
+#1-
+0x9f9e20c4 efi vmlinux EXPORT_SYMBOL
-- 
1.7.1